src/main/java/seedu/address/model/person/Meeting.java

```diff
@@ -40,41 +35,41 @@ public LocalDateTime getMeeting() {
      * Returns true if a given string is a valid meeting.
      */
     public static boolean isValidMeeting(String test) {
-        return test.matches(VALIDATION_REGEX);
+        try {
+            return test.matches(VALIDATION_REGEX) && stringToDateTime(test).isAfter(LocalDateTime.now());
+        } catch (IllegalArgumentException e) {
+            return false;
+        }
     }
 
     /**
      * This method converts a string into a LocalDateTime object.
-     * If the string is in the format "yyyy-MM-dd", it will be treated as a date at the start of the day.
-     * If the string is in the format "yyyy-MM-ddTHH:mm:ss", it will be treated as a date with time.
-     * Otherwise, the string is expected to be in the format "yyyy-MM-dd HH:mm:ss".
+     * The string must be in the format "yyyy-MM-dd HH:mm".
```

Comment:
Not having to include the seconds during input is certainly more intuitive. Good work!